code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -523,52 +522,6 @@ var/global/list/robot_modules = list(
 	FTD.synths = list(plasteel)
 	src.modules += FTD
 
-/obj/item/weapon/robot_module/security
-	name = "security robot module"
-	channels = list("Security" = 1)
-	networks = list(NETWORK_SECURITY)
-	subsystems = list(/mob/living/silicon/proc/subsystem_crew_monitor)
-	can_be_pushed = 0
-	supported_upgrades = list(/obj/item/borg/upgrade/tasercooler,/obj/item/robot_parts/robot_component/jetpack)
-
-/obj/item/weapon/robot_module/security/general
-	sprites = list(
-					"Basic" = "robotsecy",
-					"Sleek" = "sleeksecurity",
-					"Black Knight" = "securityrobot",
-					"Bloodhound" = "bloodhound",
-					"Bloodhound - Treaded" = "treadhound",
-					"Drone" = "drone-sec",
-					"Classic" = "secborg",
-					"Spider" = "spidersec",
-					"Heavy" = "heavysec"
-				)
-
-/obj/item/weapon/robot_module/security/general/New()
-	..()
-	src.modules += new /obj/item/weapon/crowbar(src)
-	src.modules += new /obj/item/device/flash(src)
-	src.modules += new /obj/item/borg/sight/hud/sec(src)
-	src.modules += new /obj/item/weapon/handcuffs/cyborg(src)
-	src.modules += new /obj/item/weapon/melee/baton/robot(src)
-	src.modules += new /obj/item/weapon/gun/energy/taser/mounted/cyborg(src)

```

Comment:
I am pretty sure this makes taser and tasercooler upgrade unused. It should be removed too.

Reply:
No, your BST borg uses it

Reply:
Then upgrade is unused.

Reply:
True. Remove the upgrade. Keep the taser itself

Reply:
We as developers decided that It's not necessary to keep cyborg taser module. Please remove it and it's reference replace with regular taser, with it's power source with infinite power cell.